gradle/defaults-java.gradle

```diff
@@ -25,13 +25,13 @@ allprojects {
     tasks.withType(JavaCompile) {
       options.encoding = "UTF-8"
       options.compilerArgs += [
-        "-Xlint", 
```

Comment:
Are these differences on EOLs? I think they've been normalized recently so it should be LF.

Reply:
OK, I'll check. I don't even know how they got changed frankly, I'll revert

Reply:
Done.